content/docs/architecture/auth.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Authentication and Authorization
+weight: 60
+---
+
+
+Fluvio uses mTLS (mutual TLS) for authentication between client and the fluvio cluster.   The user id (or principle) is extracted from the client certificate then apply to authorization policy.  Currently, Fluvio supports a simple RBAC (role based access control) authorization model but it could be extended to support more complex authorization scheme in the future. 
```

Comment:
<s>The user id (or principle) is extracted from the client certificate then apply to authorization policy. The user id (or principle) is extracted from the client certificate then apply to authorization policy. Currently, Fluvio supports a simple RBAC (role based access control) authorization model but it could be extended to support more complex authorization scheme in the future.</s> 

When the user logs in, the client downloads a profile and extracts the user identification from the client certificate issued by the server. The authorization is driven by simple RBAC (role-based access control), which can be extended for other more advanced authorization schemes in the future.

Reply:
k